diff --git a/app/controllers/files_controller.rb b/app/controllers/files_controller.rb
index 82261df..b3e425b 100644
--- a/app/controllers/files_controller.rb
+++ b/app/controllers/files_controller.rb
@@ -128,7 +128,7 @@ class FilesController < ApplicationController
   # brand-config-uploaded JS to work
   # verify_authenticity_token is manually-invoked where @context is not
   # an Account in show_relative
-  protect_from_forgery :except => [:api_capture, :show_relative], with: :exception
+  protect_from_forgery :except => [:api_capture, :show_relative, :show], with: :exception
 
   before_action :require_user, only: :create_pending
   before_action :require_context, except: [
